ostorlab_insecure_flutter_app/pubspec.yaml

```diff
@@ -36,11 +35,15 @@ dependencies:
   build_runner: any
   built_value_generator: any
 
+
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
   convert: ^3.1.1
   local_auth: ^2.1.6
+  receive_sharing_intent: ^1.4.5
```

Comment:
receive_sharint ... is not used.

Reply:
[a91d72b](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/22/commits/a91d72b4f0fa47390729212bd4c4f3fe2b828a37)